docs/src/pages/careers/lead-designer.md

```diff
@@ -0,0 +1,89 @@
+# Lead Designer
+
+<p class="description">You will own most of MUI's design work and manage a small team of designers.</p>
```

Comment:
New description: Oversee the current design team of two and manage the design work across Core, X, Toolpad, and Design Kits teams.

Reply:
I think that the current description place what matters most first: the primary goal is to own most of MUI's design work. Overseeing a team is one aspect of reaching this end. Maybe?

```suggestion
<p class="description">You will be accountable for MUI's design work and manage a small team of designers.</p>
```